content/docs/codebase-overview.md

```diff
@@ -82,19 +82,19 @@ invariant(
 );
 ```
 
-**The invariant is thrown when the `invariant` condition is `false`.**
+**El invariante se muestra cuando la condición `invariant` es `false`.**
 
-"Invariant" is just a way of saying "this condition always holds true". You can think about it as making an assertion.
+"Invariante" es una forma de decir "está condición siempre es true". Puedes pensar al respecto como si hicieras un comprobación.
 
-It is important to keep development and production behavior similar, so `invariant` throws both in development and in production. The error messages are automatically replaced with error codes in production to avoid negatively affecting the byte size.
+Es importante mantener el comportamiento del ambiente de producción y desarrollo similar, de forma que la condición `invariant` aplique para ambos. Los mensajes de error son reemplazados automáticamente por códigos de error en producción para evitar que afecten de forma negativa el tamaño en bytes.
```

Comment:
I suggest `Es importante mantener similar el comportamiento del ambiente de producción y desarrollo...`

Reply:
Yeah it is better that way.